src/city_score/city.py

```diff
@@ -58,11 +59,10 @@ def get_cities():
     with Core.open('cities.csv') as f:
         city_reader = csv.DictReader(f)
         for city_row in city_reader:
-            city = City(city_row['CITY'], city_row['STATE_CODE'], float(city_row['LATITUDE']), float(city_row['LONGITUDE']))
+            city = City(city_row['CITY'], city_row['STATE_CODE'], float(city_row['LATITUDE']), float(city_row['LONGITUDE']), city_row['STATE_NAME'])
```

Comment:
url format expects state to use standard name. lmk if there's an alternative way to retrieve this info

Reply:
@grelas Good with introducing the property. It's not a bad implementation, but it's not memory efficient (creates an additional 28k strings).

I'd insert a dictionary mapping of the states and then:

```python
state_names = {"AL":"Alabama" ... }

class City:
    @property
    def state_name(self):
        return state_names[self.state]
```

(There are a gazillion websites with the states as JSON [[example](https://usastatescode.com/state-array-json)].)